src/hera/toleration.py

```diff
@@ -20,14 +20,12 @@ class Toleration(BaseModel):
         The operator to use when matching the key to the value.
     value: str
         The value to scan for matching against a taint.
-
-    return V1Toleration(key='nvidia.com/gpu', effect='NoSchedule', operator='Equal', value='present')
     """
 
     key: str
     effect: str
     operator: str
-    value: Optional[str] = None
+    value: Optional[str] = ""
```

Comment:
Is there a reason for using a string default rather than a None?

Reply:
Default changes from `None` to `""` are motivated by the mechanism of initialization on the new objects. The generated `Toleration` object expects a `str` type for the `value` field. For `Toleration`, `value` is an _optional_ field but expects a specific field value type - optional `value` but the actual value must be a `str`. Perhaps it might be better to check for `None` and not set the field at all?